spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/SpringTestMethodBodyBuildersSpec.groovy

```diff
@@ -1934,6 +1939,41 @@ World.'''"""
 			"webclient"       | { configProperties.testMode = TestMode.WEBTESTCLIENT }
 	}
 
+	@Issue('1630')
+	def 'should generate proper test code with top level map of sets using #methodBuilderName'() {
+		given:
+		Contract contractDsl = Contract.make {
+			description('get map')
+			name('assert map response')
+			request {
+				method 'GET'
+				urlPath '/some-path'
+			}
+			response {
+				status OK()
+				body(Map.of("key", Set.of("value1", "value2")))
```

Comment:
Change this to `body(["key": ["value1", "value2"] as Set])`

Reply:
Nice shot! And a good improvement of my `groovy` skills. 
Fixed, thank you!